src/main/java/io/split/android/client/service/synchronizer/SynchronizerImpl.java

```diff
@@ -180,8 +182,11 @@ private void setupListeners() {
     }
 
     public void pause() {
-        mTaskExecutor.pause();
+        // saveImpressionsCount task is sent on pause, but it could be executed
+        // when task executor resumes when app becomes active
+        //saveImpressionsCount();
```

Comment:
shouldn't this line be uncomment? if not, `saveImpressionsCount` is never used.

Reply:
Yup doc. This going to be uncommented in next PR